stdlib/LinearAlgebra/src/diagonal.jl

```diff
@@ -448,6 +435,7 @@ function ldiv!(D::Diagonal{T}, V::AbstractMatrix{T}) where {T}
     end
     V
 end
+
```

Comment:
Compared to the code below, the order of the i- and j-loops is different. That means, the operations on `B` are performed row-wise, which is worse than column-wise in most cases. Performance checks with wide `B` required.

Reply:
I agree. We should make the `StridedVecOrMat` version be the `AbstractVecOrMat` versionn.